model/src/main/java/org/jboss/pnc/model/utils/ContentIdentityManager.java

```diff
@@ -45,9 +46,10 @@ public static String getProductContentId(ProductVersion productVersion) {
                 safeIdPart(productVersion.getVersion()));
     }
 
-    public static String getBuildContentId(String identifier) {
+    public static String getBuildContentId(BuildConfigurationAudited buildConfigurationAudited) {
         String timestamp = generateTimestamp();
-        return String.format(BUILD_CONTENT_ID_FORMAT, safeIdPart(identifier), timestamp);
+        return String.format(BUILD_CONTENT_ID_FORMAT, safeIdPart(buildConfigurationAudited.getName()), timestamp,
+                buildConfigurationAudited.getId(), buildConfigurationAudited.getRev());
```

Comment:
I think adding ID into the generated string is not necessary since name is unique (in combination with active). This makes the generated build content id unnecessarily longer, not by much, but still...

Reply:
Right, I'll change it.